src/ngraph/runtime/gpu/cudnn_emitter.cpp

```diff
@@ -1403,6 +1403,69 @@ size_t runtime::gpu::CUDNNEmitter::build_batchnorm(const cudnnBatchNormMode_t& b
     return this->m_primitive_emitter->register_primitive(batchnorm, hash);
 }
 
+size_t runtime::gpu::CUDNNEmitter::build_lrn(const cudnnLRNMode_t& lrn_op,
+                                             const std::string& dtype,
+                                             const Prop& direction,
```

Comment:
Is `direction` been used here. do we have an `backward op` need to implement later?

Reply:
Not used yet. Plumbed in for future support. Couldn't find any existing support of backwards direction in CPU backend.